docs/howto/add_pagination_links.md

```diff
@@ -67,34 +54,50 @@ ex:
 }
 ```
 
-ActiveModelSerializers pagination relies on a paginated collection with the methods `current_page`, `total_pages`, and `size`, such as are supported by both [Kaminari](https://github.com/amatsuda/kaminari) or [WillPaginate](https://github.com/mislav/will_paginate).
-
+ActiveModelSerializers pagination relies on paginated collections which define the methods `#current_page`, `#total_pages`, and `#size`.
+Such methods are supported by both [Kaminari](https://github.com/amatsuda/kaminari) or [WillPaginate](https://github.com/mislav/will_paginate),
+but you can also roll out your own paginated collection by defining these methods.
```

Comment:
At work we use neither `Kaminari` nor `WillPaginate` to paginate, but having automatic links is nice. We don't use this feature for now, but I plan to try it out soon. Therefore I would like to simply define a lightweight wrapper `PaginatedCollection.new(actual_collection, current_page, total_pages, size)` that would define these three methods and render the links. Does that make sense?